framework/tlvf/tlvf_conf.yaml

```diff
@@ -10,30 +10,31 @@ include_yaml_path: {
   # "wfa_map/",
   "tlvf/wfa_map/eTlvTypeMap.yaml",
   "tlvf/wfa_map/tlvApCapability.yaml",
-  "tlvf/wfa_map/tlvChannelPreference.yaml",
-  "tlvf/wfa_map/tlvSupportedService.yaml",
+  "tlvf/wfa_map/tlvApHeCapabilities.yaml",  
```

Comment:
This commit should be split into 2 commits:
1. Reorder the lines alphabetically
2. Add your yaml file

Reply:
👍